src/phoenix/datetime_utils.py

```diff
@@ -49,7 +49,7 @@ def normalize_timestamps(
             timezone.utc,
         )
     if is_object_dtype(timestamps):
-        timestamps = to_datetime(timestamps)
+        timestamps = to_datetime(timestamps, format="ISO8601")
```

Comment:
I might be mistaken but this might break some of our flows as they may not be in ISO08601 format? It would be a lot safer if we detected that the timestamps were in this format before assuming the format here. Could you add a test to ensure this doesn't break for other formats?

@RogerHYang might know more - I might be completely off here.

Reply:
@mikeldking 
- Is it really a good thing that we might have different time formats in our codebase? 
- To drastically simplify things, we can add a parameter that specifies the format. We can drill it down and pass it instead of hardcoding it `ISO8601`. So when someone get's this error, they can pass any format they way. We can possibly limit to some of the ISO formats.
- You are right, adding a test does sounds good, even if we go with the parameter approach, we can still make sure the normalization flow is airtight.